core/src/franchise/player/player.controller.ts

```diff
@@ -36,122 +38,127 @@ export class PlayerController {
 
-        await this.eventsService.publish(EventTopic.PlayerSkillGroupChanged, {
-            playerId: player.id,
-            name: player.member.profile.name,
-            organizationId: player.skillGroup.organizationId,
-            discordId: discordAccount.accountId,
-            old: {
-                id: player.skillGroup.id,
-                name: player.skillGroup.profile.description,
-                salary: Number(player.salary),
-                discordEmojiId: player.skillGroup.profile.discordEmojiId,
-            },
-            new: {
-                id: skillGroup.id,
-                name: skillGroup.profile.description,
-                salary: Number(payload.salary),
-                discordEmojiId: skillGroup.profile.discordEmojiId,
-            },
-        });
+            await this.eventsService.publish(EventTopic.PlayerSkillGroupChanged, {
+                playerId: player.id,
+                name: player.member.profile.name,
+                organizationId: player.skillGroup.organizationId,
+                discordId: discordAccount.accountId,
+                old: {
+                    id: player.skillGroup.id,
+                    name: player.skillGroup.profile.description,
+                    salary: Number(player.salary),
+                    discordEmojiId: player.skillGroup.profile.discordEmojiId,
+                },
+                new: {
+                    id: skillGroup.id,
+                    name: skillGroup.profile.description,
+                    salary: Number(payload.salary),
+                    discordEmojiId: skillGroup.profile.discordEmojiId,
+                },
+            });
 
-        await this.notificationService.send(NotificationEndpoint.SendNotification, {
-            type: NotificationType.BASIC,
-            userId: player.member.user.id,
-            notification: {
-                type: NotificationMessageType.DirectMessage,
-                userId: discordAccount.accountId,
-                payload: {
-                    embeds: [ {
-                        title: "You Have Ranked Out",
-                        description: `You have been ranked out from ${player.skillGroup.profile.description} to ${skillGroup.profile.description}.`,
-                        author: {
-                            name: `${orgProfile.name}`,
-                        },
-                        fields: [
-                            {
-                                name: "New League",
-                                value: `${skillGroup.profile.description}`,
+            await this.notificationService.send(NotificationEndpoint.SendNotification, {
+                type: NotificationType.BASIC,
+                userId: player.member.user.id,
+                notification: {
+                    type: NotificationMessageType.DirectMessage,
+                    userId: discordAccount.accountId,
+                    payload: {
```

Comment:
While you're in here making changes, break out this payload into another function like `buildRankdownNotification` or something. This function is really long